base/applications/mspaint/main.cpp

```diff
@@ -340,7 +340,7 @@ _tWinMain (HINSTANCE hThisInstance, HINSTANCE hPrevInstance, LPTSTR lpszArgument
     mainWindow.SetWindowPlacement(&(registrySettings.WindowPlacement));
 
     /* Make the window visible on the screen */
-    ShowWindow (hwnd, nFunsterStil);
```

Comment:
You may replace as well "nFunsterStil" in the _tWinMain prototype above:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=nFunsterStil
by `nShowCmd`  (standard name).

Reply:
Renamed as `nCmdShow`.